src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -87,6 +87,9 @@ public static class ApplicationInsightsTelemetry
         // the session identifier
         private static string s_sessionId { get; set; }
 
+        // private semaphore to determine whether we sent the startup telemetry event
+        private static bool s_startupEventSent { get; set; }
```

Comment:
should this just be a field?

Reply:
Looks like the other members are all properties.  We can decide to change them as fields in a separate PR if needed.